spec/bolt/module_installer/puppetfile_spec.rb

```diff
@@ -112,5 +112,20 @@
         )
       end
     end
+    context 'with missing module version' do
+      let(:spec) { double('spec', satisfied_by?: true)}
+      let(:version) {nil}
```

Comment:
```suggestion

    context 'with missing module version' do
      let(:spec)    { double('spec', satisfied_by?: true) }
      let(:version) { nil }
```

Changes to whitespace for improved readability / consistency with other tests.